apps/web/src/components/DataSourcesList.tsx

```diff
@@ -92,6 +99,8 @@ const databaseUrl = (ds: DataSource): string => {
         return `oracle://${ds.data.host}:${ds.data.port}/${ds.data.database}`
       case 'mysql':
         return `mysql://${ds.data.host}:${ds.data.port}/${ds.data.database}`
+      case 'sqlserver':
+        return `sqlserver://${ds.data.host}:${ds.data.port};databaseName=${ds.data.database};user=${ds.data.username};password=******;`
```

Comment:
I think we can only include this. Adding the `*` for Athena was probably a mistake on our end.

```suggestion
        return `sqlserver://${ds.data.host}:${ds.data.port};databaseName=${ds.data.database};`
```